impeller/typographer/glyph.h

```diff
@@ -15,9 +15,19 @@ namespace impeller {
 /// @brief      The glyph index in the typeface.
 ///
 struct Glyph {
+  enum class Type {
+    kPath,
+    kBitmap,
+  };
+
   uint16_t index = 0;
 
-  Glyph(uint16_t p_index) : index(p_index) {}
+  //------------------------------------------------------------------------------
+  /// @brief      Whether the glyph is a path or a bitmap.
+  ///
+  Type type = Type::kPath;
```

Comment:
This should be incorporated into equality/hash right?

Reply:
I... guess? This is specific to the glyph ID though.

Reply:
yeah....I dunno. As long as we never have glyph collisions its fine